packages/daemon/src/daemon.js

```diff
@@ -78,6 +78,11 @@ const getDerivedId = (path, rootNonce, digester) => {
   return nonce;
 };
 
+/**
+ * @template T
+ * @typedef {import('./types.js').IncarnateResult<T>} IncarnateResult
+ */
```

Comment:
Let’s inline these. Typedefs create new types and interrupt “go to definition” chains.